src/components/form-input-wrapper.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div :class="[valid ? 'is-valid' : '', invalid ? 'is-invalid' : '']">
+    <CFormLabel v-if="label">{{ label }}</CFormLabel>
```

Comment:
we need to support other html attribute like placeholder, required to show * etc